app/src/packages/editor/hooks/useColorMode.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+import useLocalStorage from './useLocalStorage';
+
+export type ThemeMode = 'light' | 'dark' | 'system';
+
+export default function useColorMode() {
+  const [colorMode, setColorMode] = useLocalStorage<ThemeMode>('color-theme', 'light');
```

Comment:
packages/editor uses jotai to manage global states, see the other definitions in atoms.ts.
Here I don't recommend using useLocalStorage, but instead, you should define a jotai atom to mange it.